lib/list.js

```diff
@@ -0,0 +1,85 @@
+const Id = require('hypercore-id-encoding')
+const c = require('compact-encoding')
+const Hyperbee = require('hyperbee')
+const same = require('same-object')
+
+const types = ['core', 'bee', 'drive']
+
+module.exports = class List {
+  constructor (core) {
+    this.core = core
+    this.bee = new Hyperbee(core, { keyEncoding, valueEncoding })
+  }
+
+  ready () { return this.bee.ready() }
```

Comment:
Could be a ready-resource for simplicity

Reply:
That adds a lib, plus makes it an emitter, plus still needs to forward both bee ready + close calls, sounds overkill

Reply:
I do think it needs logic to close the bee, because `List` objects will be managing the lifecycle of the passed-in hypercore

Reply:
Yep you're right